cg/cli/demultiplex/copy_novaseqx_data.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+import os
+import shutil
+from typing import List, Optional
+
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+def is_demultiplexing_copied(analysis_directory: Path) -> bool:
+    """Determine whether the demultiplexing has been copied for the latest analysis for a novaseqx flow cell."""
+    return Path(analysis_directory, DemultiplexingDirsAndFiles.COPY_COMPLETE).exists()
+
+
+def is_flow_cell_demultiplexed(analysis_directory: Path) -> bool:
+    return Path(
+        analysis_directory,
+        DemultiplexingDirsAndFiles.DATA,
+        DemultiplexingDirsAndFiles.ANALYSIS_COMPLETED,
+    ).exists()
+
+
+def is_flow_cell_in_demultiplexed_runs(flow_cell_name: str, demultiplexed_runs: Path) -> bool:
+    return Path(demultiplexed_runs, flow_cell_name).exists()
+
+
+def get_latest_analysis_directory(flow_cell_dir: Path) -> Optional[Path]:
+    analysis_path = Path(flow_cell_dir, DemultiplexingDirsAndFiles.ANALYSIS)
+
+    if not analysis_path.exists():
+        return None
+    analysis_versions = get_sorted_analysis_versions(analysis_path)
```

Comment:
```suggestion
    analysis_version_paths: List[Paths] = get_sorted_analysis_versions(analysis_path)
```

Reply:
One could argue that the type hint does not add any new information either since it already is given in the signature of `get_sorted_analysis_versions`